extensions/amp-analytics/0.1/requests.js

```diff
@@ -92,6 +116,11 @@ export class RequestHandler {
   send(configParams, trigger, expansionOption) {
     this.lastTrigger_ = trigger;
     const triggerParams = trigger['extraUrlParams'];
+    const batchSegment = dict({
```

Comment:
why dict?

Reply:
Hmmm, I really like `dict` function : ) it can create a `JsonObject`